src/sidebar/store/modules/real-time-updates.js

```diff
@@ -140,7 +140,7 @@ function clearPendingUpdates() {
  * Return added or updated annotations received via the WebSocket
  * which have not been applied to the local state.
  *
- * @return {{[id: string]: Annotation}}
+ * @return {Object<string, boolean>}
```

Comment:
I took the liberty of making this change because I want to stick with one standard or another here. I was not sure if the existing type was correctly formatted or not. I could use some input.

Reply:
I'm fine with this change. Both mean exactly the same but we've been favouring the second style. I find the `Object.<KeyType, ValueType>` syntax slightly more readable in JSDoc `{...}` parens. A small downside is that you can't name the key parameter.